### ISSUE TITLE
Update code documentation template

### ISSUE THREAD

**Issue by mattrosno** [carbon-design-system/carbon-platform#817]

**Comment by mattrosno** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-624ca8ad03d709001dc542bf/issues/carbon-design-system/carbon-platform/817?planning-poker) with ZenHub @aubrey-oneal @kingtraceyj

**Comment by jeanservaas** [comment]
@mattrosno @alisonjoseph I don't think we need a code tab now. We have no dev docs on it for the most part for any carbon components. When we start to add docs, or content starts coming in from PALs, we can make a case to add it. But I'd like to avoid a bunch of empty pages until we have content.

**Comment by mattrosno** [comment]
@jeanservaas we need a code tab. A lot of the PAL components and patterns have code tabs. Yes some just link to Storybook, but others show code status tables and detailed documentation.

- https://pages.github.ibm.com/ai-applications/design/components/color-picker/code
- https://pages.github.ibm.com/cdai-design/cloud-pal/patterns/email-templates/code
- https://pages.github.ibm.com/cdai-design/cloud-pal/patterns/provisioning/code
- https://pages.github.ibm.com/cdai-design/cloud-pal/layouts/world-level-layout/code

**Comment by mattrosno** [comment]
> But I'd like to avoid a bunch of empty pages until we have content.

@jeanservaas if you don't want minimal code tabs for Carbon React components, then we don't index a `codePath` for those components, and the asset detail pages for those components will have no code tab.

**Comment by jeanservaas** [comment]
Sounds good on the code tab.

**Comment by mattrosno** [comment]
@jeanservaas sounds great I'm going to remove the estimation and design label and convert this to a dev issue to do.
